docs/pages/deploy-a-cluster/teleport-enterprise/getting-started.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Teleport Enterprise Quick Start
 description: How to set up and configure Teleport Enterprise for SSH
-h1: Teleport Enterprise Quick Start
+# Teleport Enterprise Quick Start
```

Comment:
Is this necessary? It's the same as the title, so can we just remove it?

Reply:
Not to my knowledge, and I can see others removing it entirely within our repo. Removing as well